contrib/fonttools/showttf.c

```diff
@@ -2887,6 +2887,7 @@ static void gdefshowligcaretlist(FILE *ttf,int offset,struct ttfinfo *info) {
 		printf( "!!!! Unknown caret format %d !!!!\n", format );
 	    }
 	}
+        free(offsets);
```

Comment:
git push -f will overwrite what you previously pushed (make sure you don't loose anything in the process).